src/assets/scss/components/dashboard.scss

```diff
@@ -62,4 +62,9 @@ $sectionAreaSpacing: 15px;
   .pf-c-button {
     color: black;
   }
-}
\ No newline at end of file
+}
+
+@keyframes newRowHighlightAnimation {
```

Comment:
Use `kebab-case` for CSS names

Reply:
Done: https://github.com/darwinai/covidnet_ui/pull/19/commits/b4238759c2a1bf6549364f17c544dca92105f06f